virt/virt-web-console-overview.adoc

```diff
@@ -920,6 +923,36 @@ Click the Options menu {kebab} beside a snapshot to select *Edit labels*, *Edit
 |====
 =====
 
+[id="ui-virtualmachine-details-diagnostics_{context}"]
+==== Diagnostics tab
+
+You can view the status conditions and volume snapshot status on the *Diagnostics* tab.
+
+.Diagnostics tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element
+|Description
+
+|*Status conditions* table
+|Display a list of conditions that are reported for all aspects of a VM.
```

Comment:
I guess technically this is the first appearance of VM in the assembly to this point (only saw one other instance, down in "Parameters tab"), so _might_ need the full spelled-out "virtual machine (VM)" per [RH SG](https://redhat-documentation.github.io/supplementary-style-guide/#virtual-machine)?